raiwidgets/requirements-dev.txt

```diff
@@ -28,4 +28,5 @@ nbval
 
 sphinx==3.1.1
 sphinx-gallery==0.8.1
-pydata-sphinx-theme==0.3.0
\ No newline at end of file
+pydata-sphinx-theme==0.3.0
+../responsibleai/.
```

Comment:
Does this actually work? Perhaps just saying "responsibleai" would be fine?

Reply:
I have to change pipeline to run inside this folder.
if we use responsibleai it will install from pypi, which will not include local changes of responsibleai

Reply:
Isn't that how it should be? Because if you run a release of raiwidgets then it will depend on the responsibleai version from pypi, not the repo version. Also, changes to packages should be to one package only, and otherwise split over multiple PRs. Wdyt?